snaps_k8s/ansible_p/commission/kubernetes/playbooks/deploy_mode/k8/set_hostname.yaml

```diff
@@ -0,0 +1,41 @@
+#* Copyright 2018 ARICENT HOLDINGS LUXEMBOURG SARL and Cable Television
+# Laboratories, Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This script is responsible for deploying Aricent_Iaas environments and
+# Kubernetes Services
+---
+
+- hosts: all
+  become: yes
+  become_method: sudo
+  become_user: root
+
+  tasks:
+    - name: change host name to "{{host_name}}"
+      command: hostname "{{host_name}}"
+
+    - name: remove file /etc/hostname
+      file:
+        dest: /etc/hostname
```

Comment:
Why is /etc/hostname being modified?

It may be good to update /etc/hosts

Reply:
Old logic moved out of setup_k8.yaml that we can change later...